src/app/shared/learning-object/learning-object.component.ts

```diff
@@ -47,25 +48,12 @@ export class LearningObjectListingComponent implements OnInit, OnChanges {
 
     this.collectionService.getCollections().then(collections => {
       this.collections = new Map(collections.map(c => [c.abvName, c.name] as [string, string]));
+    }).catch(error => {
     });
   }
 
   goals() {
     const punc = ['.', '!', '?'];
-    // convert goals to an array of strings containing the goal text
```

Comment:
Why were the comments removed? This still seems to be an operation that isn't quite straightforward.

Reply:
These comments were removed because most of the code here wasn't used. The deleted blob was a collection of unused code and comments pertaining to the unused code.